docker-dd-agent-no-checks/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM datadog/docker-dd-agent:12.7.5327
+
+ARG DOCKER_VERSION=12.7.5327
+
+LABEL version="${DOCKER_VERSION}"
+LABEL maintainer="sakamoto@chatwork.com"
+
+RUN rm /etc/dd-agent/conf.d/docker_daemon.yaml
+RUN rm /etc/dd-agent/conf.d/*.default
```

Comment:
@cw-sakamoto 
Tell me about it. Are you deleting the configuration file because this can't be controlled by environment variables?

Reply:
https://github.com/everpeace/docker-dd-agent-no-checks/blob/master/Dockerfile

Sorry, I originally used ↑ image only to send metrics to datadog inside burrow by sidecar, and I don't know why it is such a configuration...😱

It's easier to manage in our own repository, so I brought it in as is.

Reply:
OK, I understand.

Reply:
@cw-sakamoto 
Consider making it an original docker image in the future.
I think I can do that.